doc/how-to/perform-a-release.md

```diff
@@ -0,0 +1,43 @@
+# Perform a release
+
+## Prerequisites
+
+- Be a part of the Ministry of Justice GitHub organisation. This should be done
+  as part of your your team onboarding process.
+- Be a part of the `accredited-programmes-team` team. This should be done as
+  part of your team onboarding process.
+
+## Releasing to the development environment
+
+This will be performed automatically on a merge to the `main` branch providing
+the `integration_test`, `build_docker`, `helm_lint` and `unit_test` jobs pass on
+CI.
+
+## Releasing to the preprod environment
+
+Once the `deploy_dev` CI job has passed on CI after merging a branch to `main`,
+the pipeline will hold and await manual approval (as documented in the Circle CI
+documentation
+[here](https://circleci.com/docs/workflows/#holding-a-workflow-for-a-manual-approval))
+at the `deploy_preprod` step.
+
+Perform any necessary manual testing on the development environment and ensure
+your changes are backwards compatible.
+
+Clicking on that step of the workflow and selecting "Approve" will deploy the
+latest version of the code to the preprod environment.
+
+## Releasing to the production environment
+
+Once the `deploy_preprod` CI job has passed on CI after performing the above
+step, the pipeline will hold again and await manual approval at the
+`deploy_prod` step.
+
+Clicking on that step of the workflow and selecting "Approve" will deploy the
+latest version of the code to the production environment.
+
+## Rolling back a release
```

Comment:
If you need to roll back a change urgently you can do this using the Helm command line client.  
This is quick and buys you time to back out your change (or fix forward) via GitHub and CircleCI.

Get the deployment history:
```
helm history hmpps-accredited-programmes-ui
```
This gives you something like
```
REVISION	UPDATED                 	STATUS    	CHART                               	APP VERSION            	DESCRIPTION
72      	Tue Jun 20 08:56:16 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-20.2073.b0e85e7	Upgrade complete
73      	Tue Jun 20 09:32:12 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-20.2084.3202b60	Upgrade complete
74      	Tue Jun 20 13:50:49 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-20.2104.87bff67	Upgrade complete
75      	Tue Jun 20 14:10:53 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-20.2118.625021e	Upgrade complete
76      	Wed Jun 21 11:06:00 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-21.2148.916ee09	Upgrade complete
77      	Wed Jun 21 15:58:11 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-21.2168.f4239d9	Upgrade complete
78      	Wed Jun 21 16:07:28 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-21.2185.249395f	Upgrade complete
79      	Thu Jun 22 10:32:17 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-22.2240.dd4d7ae	Upgrade complete
80      	Thu Jun 22 13:10:02 2023	superseded	hmpps-accredited-programmes-ui-0.2.0	2023-06-22.2257.5aa23b4	Upgrade complete
81      	Thu Jun 22 13:31:16 2023	deployed  	hmpps-accredited-programmes-ui-0.2.0	2023-06-22.2274.2d86b57	Upgrade complete
```
Then roll back to a previous deployment. Give the revision number to roll back to:
```
helm rollback hmpps-accredited-programmes-ui 78
```